sql/telemetry_derived/amo_stats_dau_v1/query.sql

```diff
@@ -0,0 +1,189 @@
+/*
+
+Daily user statistics to power AMO stats pages. See bug 1572873.
+
+Each row in this table represents a particular addon on a particular day
+and provides all the information needed to populate the various
+"Daily Users" plots for the AMO stats dashboard.
+
+*/
+--
+WITH cd AS (
+  SELECT
+    submission_date,
+    client_id,
+    active_addons,
+    app_version,
+    country,
+    locale,
+    os
+  FROM
+    telemetry.clients_daily
+  WHERE
+    ARRAY_LENGTH(active_addons) > 0
+    AND submission_date = @submission_date
+),
+--
+unnested AS (
+  SELECT
+    * EXCEPT (active_addons, version, os),
+    version AS addon_version,
+    os AS app_os,
```

Comment:
curious: why rename this here rather than in the `cd` expression?

Reply:
That sounds reasonable. Will change.